projects/ids-enterprise-ng/src/lib/stepprocess/soho-stepprocess.component.ts

```diff
@@ -73,7 +74,17 @@ export class SohoStepListItemComponent {
   @HostBinding('class.js-step') get jsStep() {
     return true;
   }
-  @HostBinding('class.is-selected') @Input() isSelected = false;
+
+  @HostBinding('class.is-selected') @Input() set isSelected(isSelected: boolean) {
+    if (isSelected) {
+      const stepLink = $(this.element.nativeElement).children('a.js-step-link');
+      this.sohoStepProcessComponent.stepprocess?.selectStep(stepLink);
+    }
+  }
```

Comment:
Since this becomes a setter, we'll also need a to set the value in a private property and expose it via a getter. Hostbinding for class should be moved there as well

Reply:
Agree with the above comment.

```typescript 

  @HostBinding('class.is-selected') _isSelected: boolean = false;

  @Input() set isSelected(isSelected: boolean) {
    this._isSelected = isSelected;
    if (isSelected) {
      const stepLink = $(this.element.nativeElement).children('a.js-step-link');
      this.sohoStepProcessComponent.stepprocess?.selectStep(stepLink);
    }
  }
```